swarm/pss/pss.go

```diff
@@ -179,40 +213,82 @@ func (self *Pss) Register(topic *Topic, handler Handler) func() {
 	return func() { self.deregister(topic, &handler) }
 }
 
-func (self *Pss) deregister(topic *Topic, h *Handler) {
+// Set the amount of bytes that will be disclosed encrypted of recipient address
+//
+// 0 will equal whisper routing (forward to all, no unencrypted partial address)
+// -1 will return to default value
+func (self *Pss) SetRecipientAddressLength(l int) {
```

Comment:
I think this is a bit security liability completely unnecessarily
No i think it should be transparent in the explicit length of the recipient address coming from the API
it is entirely at the discretion of the client app to manage luminosity :)